connexion/decorators/validation.py

```diff
@@ -117,7 +117,8 @@ def validate_schema(self, data):
         try:
             validate(data, self.schema, format_checker=draft4_format_checker)
         except ValidationError as exception:
-            return problem(400, 'Bad Request', str(exception))
+            logger.debug("%s validation error: %s" % (flask.request.url, exception))
+            raise
```

Comment:
Maybe better to use it https://pythonhosted.org/six/#six.reraise
So it is compatible across Python versions.